### DISCUSSION TITLE
Multi-select feature in menu and combobox

### DISCUSSION THREAD

**Discussion by leeejune** [AlaskaAirlines/WC-Generator#534] [Announcements]
As a user, I want to search and filter through a list and select multiple items.

**[Combobox blueprint](https://www.figma.com/file/yMshXAiKZrkLDdiX8N4GKQ/Multi-Select-Combo-Box?type=design&node-id=114-3745&mode=design&t=B9CA2r1WFDVeeUr7-11)**

## Questions to be answered
What component will this feature belong to? (menu, combobox, select, etc)

**Comment by blackfalcon** [comment]
## Meeting notes
* Things discussed in this review were expected interaction models, state management and hypothesized edge-cases. 
* A user may enter keywords into the text area that will filter responses in the dropdown menu
* A user may select multiple options in the menu that will be reflected by a visible checkbox 
* As items are selected, the placeholder in the input will reflect 1 item or 2 items selected. There is no expectation to list the items selected, that is what the top of the UI is designed to do. 
* Only when the user blurs away from the multi-select experience and then comes back will the user see their selections cloned to the top of the menu UI. 
* As a user unselects options from the top of the menu UI, these options will remain until the user blurs away and comes back. 
* As a user starts to enter text into the input, the selected options that appear at the top of the menu will not be visible. It's only when the input has no value that the pre-selected options appear. 
* Edge case experience was brought up about potential issues with data changing while the user is interacting with the element. E.g. what if an option is no longer available between selection and coming back to see selected options? The current answer is to simply reflect the data that is available. 

Design team was tasked with updating the Figma per meeting feedback.
